api/src/main/java/io/grpc/CallOptions.java

```diff
@@ -203,6 +211,12 @@ public CallOptions withoutWaitForReady() {
     return builder.build();
   }
 
+  public CallOptions withOnReadyThreshold(int numBytes) {
```

Comment:
This should probably be `Integer` to allow clearing the value? Or we could have a separate method that sets it to null.

Reply:
Added a clearOnReadyThreshold() method.